tests/vmcheck/test-livefs.sh

```diff
@@ -144,9 +151,7 @@ set -xeuo pipefail
 cd /ostree/repo/tmp
 rm vmcheck -rf
 ostree checkout vmcheck vmcheck --fsync=0
-(cat vmcheck/usr/bin/ls; echo more stuff) > vmcheck/usr/bin/ls.new
-chmod a+x vmcheck/usr/bin/ls.new
-mv vmcheck/usr/bin/ls{.new,}
+(date; echo "JUST KIDDING DO WHATEVER") >vmcheck/${dummy_file_to_modify}.new && mv vmcheck/${dummy_file_to_modify}{.new,}
```

Comment:
As a corner-case, maybe also `mkdir vmcheck/usr/newdir` and check that it was created later on down there?